iiify/app.py

```diff
@@ -77,6 +81,29 @@ def demo():
 def documentation():
     return render_template('docs/index.html', version=version)
 
```

Comment:
I think the detail of the method should probably be moved to resolver.py to keep it consistent with the other requests.